test/lib/extensions.js

```diff
@@ -616,7 +616,7 @@ test("Project with task extension dependency", (t) => {
 	});
 });
 
-test("Project with task extension dependency - task module not found", async (t) => {
+test("Project with task extension dependency - does not throw for invalid task path", async (t) => {
```

Comment:
As already mentioned in the other PR, what about a test when the `require` of the custom task fails? Which error does the user get?

Reply:
No error here. The require now happens when a different module (e.g. MiddlewareManager or *Builder) tries to retrieve the middleware or task. I added additional error handling there and some tests.